src/docs/development/packages-and-plugins/androidx-compatibility.md

```diff
@@ -4,18 +4,17 @@ description: How to fix AndroidX incompatibilities that have been detected by th
 ---
 
 {{site.alert.note}}
-  You might be directed to this page if the framework detects a problem in your
```

Comment:
I thought I left some comments here yesterday but I can't find it anymore. It might not have gone through.

I unfortunately have to revert the non-add-to-app changes in this add-to-app branch. Keeping 2 branches is difficult but feasible for purely additive changes. But for changes on existing files like this, it's hard to merge since the base text already changed on master (specifically for this file, it already moved on master). I tried to manually downport some of the non-add-to-app changes back to master but since the base text was so different, I think it's going to be easier to just re-create these changes directly on an up-to-date master. 

TL;DR the new plugin-api-migration.md and sidenav.yml will be mergeable to master but the rest will be difficult.